packages/app/src/client/services/AdminLocalSecurityContainer.js

```diff
@@ -30,9 +30,15 @@ export default class AdminLocalSecurityContainer extends Container {
 
   async retrieveSecurityData() {
     try {
+      // retrieve basic setting to get isMailerSetup data, validation for email authentication
+      const responseAppSetting = await this.appContainer.apiv3.get('/app-settings/');
+      const { appSettingsParams } = responseAppSetting.data;
+
       const response = await this.appContainer.apiv3.get('/security-setting/');
       const { localSetting } = response.data.securityParams;
+
       this.setState({
+        isMailerSetup: appSettingsParams.isMailerSetup, // add isMailerSetup for email authentication condition
```

Comment:
please do not set the `isMailerSetup` here because it’s unrelated to security data

Reply:
done